help/destinations/catalog/advertising/liveramp-onboarding.md

```diff
@@ -93,10 +98,11 @@ To authenticate to the destination, fill in the required fields and select **[!U
 
 To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
 
-![Platform UI screenshot showing how to fill in details for your destination](../../assets/catalog/advertising/liveramp-onboarding/liveramp-connection-details.png)
+![Platform UI screenshot showing how to fill in details for your destination](../../assets/catalog/advertising/liveramp-onboarding/liveramp-sftp-destination-details.png)
 
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Region]**: Geographic region for your instance of the LiveRamp sftp.
```

Comment:
*  **[!UICONTROL Region]**: Geographic region of your instance of the LiveRamp SFTP storage.